lnrpc/rpc_utils.go

```diff
@@ -85,16 +83,6 @@ func RPCTransactionDetails(txns []*lnwallet.TransactionDetail) *TransactionDetai
 		txDetails.Transactions[i] = RPCTransaction(tx)
 	}
 
-	// Sort transactions by number of confirmations rather than height so
-	// that unconfirmed transactions (height =0; confirmations =-1) will
-	// follow the most recently set of confirmed transactions. If we sort
-	// by height, unconfirmed transactions will follow our oldest
-	// transactions, because they have lower block heights.
-	sort.Slice(txDetails.Transactions, func(i, j int) bool {
```

Comment:
I think we can leave this here but modify it if needed by the caller.

Reply:
Oh okay thanks I will move it back there